tests/test_cpp_interface.cpp

```diff
@@ -0,0 +1,68 @@
+#include <CppUTest/TestHarness.h>
+#include <CppUTestExt/MockSupport.h>
+#include <memory>
+#include "../messagebus.h"
+
+
+TEST_GROUP(MessagebusCppInterface)
+{
+    messagebus_topic_t raw_topic;
+    int topic_content;
+    messagebus_t bus;
+    std::unique_ptr<messagebus::TopicWrapper<int> > topic;
```

Comment:
Why do you use a `unique_ptr` here instead of an instance of `TopicWrapper`?

Reply:
Because TopicWrapper does not have a default constructor, so I cannot use it as a static test variable. I do not know a better way to delay static init here.

Reply:
Do you know a better way ?

Reply:
clang disagrees:

```
external/ch_cvra_msgbus/tests/test_cpp_interface.cpp:12:42: error: C++ requires a type specifier for all declarations
    messagebus::TopicWrapper<int> topic(&raw_topic);
                                         ^
```

Reply:
It thinks you're declaring a function, you should do `messagebus::TopicWrapper<int> topic{&raw_topic};` or `messagebus::TopicWrapper<int> topic = messagebus::TopicWrapper<int>(&raw_topic);` ;)